processor/memorylimiter/testdata/config.yaml

```diff
@@ -0,0 +1,36 @@
+receivers:
+  examplereceiver:
+
+processors:
+  memory_limiter:
+    # empty config
+
+  memory_limiter/with-settings:
+    # check_interval is the time between measurements of memory usage for the
+    # purposes of avoiding going over the limits. Defaults to zero, so no
+    # checks will be performed. Values below 1 second are not recommended since
```

Comment:
We've been using 100ms for this. Do you have measurements showing noticeable CPU reduction using something like 1s?

Reply:
This uses `runtime.ReadMemStats` to read heap size, which stops the world. I didn't measure but suspect it can get expensive as the heap grows. We can measure, but I wanted to be on the safe side with the recommendation.

Reply:
Here is one anecdotal evidence: https://github.com/golang/go/issues/13613#issuecomment-267690123

>I took an execution trace from an application with a 15-30GB heap, and a call to runtime.ReadMemStats() paused it for about 40 ms.

Looks like I am not too far off. I'd say going below 1s period is risky. I'd run at 5s or higher.

Reply:
sgtm